core/tests/test_ingress_adv.py

```diff
@@ -21,7 +21,7 @@
     ERR_IMPORT_ERROR_3,
 )
 
-from crop.ingress_adv import (
+from core.crop.ingress_adv import (
```

Comment:
Instead of adding core., could you please adjust your python path since we have restructured the repository.